test/standalone/child_process/child.zig

```diff
@@ -0,0 +1,51 @@
+const std = @import("std");
+
+// 42 is expected by parent; other values result in test failure
+var exit_code: u8 = 42;
```

Comment:
the "expected" part is actually in the parent. this is a child - it doesn't check against an expected value but simply exits with an exit code. feels like it would be more confusing with "expected_" prefix.